config/environments/production.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "example.com",
+    authentication: "plain",
+    enable_starttls_auto: true,
+    user_name: ENV["GMAIL_USERNAME"],
+    password:ENV["GMAIL_PASSWORD"]
+  }
```

Comment:
I think we'll want to use Sendgrid for this kind of stuff, especially in production. In particular, we'll want to use `SENDGRID_USERNAME` and `SENDGRIND_PASSWORD`. We'll also want to `smtp.sendgrid.net` as the address, and something like `rescuingleftovercuisine.org` as the domain.

I think we should do this in staging, too, and we can sort out how to test it with .env.

Also, probably want a space after `password:`.